### ISSUE TITLE
Question about cumsum of the planning prediction 

### ISSUE THREAD

**Issue by Christinepan881** [OpenDriveLab/UniAD#92]
Hi authors, 
In the planning_head.py 191 line: `sdc_traj_all[...,:2] = torch.cumsum(sdc_traj_all[...,:2], dim=2)` , why do the cumsum along the xy axis? Why not along the future-timestep axis?

**Comment by ilnehc** [comment]
@Christinepan881 The dim 2 is along the planning steps axis (1,1,256 -> 1,1,T*2 ->1,1,T,2).
https://github.com/OpenDriveLab/UniAD/blob/fa7fc3edc0811e1aff6dc952e38478a997c0843d/projects/mmdet3d_plugin/uniad/dense_heads/planning_head.py#L188

**Comment by Christinepan881** [comment]
> @Christinepan881 The dim 2 is along the planning steps axis (1,1,256 -> 1,1,T*2 ->1,1,T,2).
> 
> https://github.com/OpenDriveLab/UniAD/blob/fa7fc3edc0811e1aff6dc952e38478a997c0843d/projects/mmdet3d_plugin/uniad/dense_heads/planning_head.py#L188

But the tensor is reshaped to [1, T, 2] in the 190 line, which means the cumsum is performed along the xy axis (dim 2): 
https://github.com/OpenDriveLab/UniAD/blob/fa7fc3edc0811e1aff6dc952e38478a997c0843d/projects/mmdet3d_plugin/uniad/dense_heads/planning_head.py#L190

**Comment by ilnehc** [comment]
@Christinepan881 Thank you for reporting the issue. We have updated the code (7903eadbcf5a5592b90990c02ab9ff15ba332129) and the pretrained weight for stage 2 (https://github.com/OpenDriveLab/UniAD/releases/tag/v1.0.1).

**Comment by pac2syv** [comment]
@ilnehc Thanks for the updating! The 'pretrained weight' means pretrained weights for stage-2 training initialization or the weights after stage-2 training?

**Comment by ilnehc** [comment]
@pac2syv It is the checkpoint after stage-2 training.
